Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-results-grid/ContestResultsGrid.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Tooltip } from '@mui/material';
+import isEmpty from 'lodash/isEmpty';
+import isNil from 'lodash/isNil';
+
+import { IContestResultsParticipationType, IContestResultsType } from '../../../hooks/contests/types';
+import { useContestCategories } from '../../../hooks/use-contest-categories';
+import useTheme from '../../../hooks/use-theme';
+import { IAuthorizationReduxState } from '../../../redux/features/authorizationSlice';
+import isNilOrEmpty from '../../../utils/check-utils';
+import concatClassNames from '../../../utils/class-names';
+import { ButtonSize, LinkButton, LinkButtonType } from '../../guidelines/buttons/Button';
+
+import styles from './ContestResultsGrid.module.scss';
+
+interface IContestResultsGridProps {
+    items: IContestResultsType | null;
+}
+
+const ContestResultsGrid = ({ items }: IContestResultsGridProps) => {
+    const { state: { categoriesFlat }, actions: { load: loadCategories } } = useContestCategories();
```

Comment:
use data from slice and get data from services, we should remove all these hooks